src/Test.FunctionalTests.BinSkim.Driver/BaselineTestData/Expected/ManagedInteropAssemblyForAtlTestLibrary.dll.sarif

```diff
@@ -315,7 +315,28 @@
       },
       "invocations": [
         {
-          "executionSuccessful": true
+          "toolConfigurationNotifications": [
```

Comment:
>toolConfigurationNotifications [](http://example.com/codeflow?start=11&length=30)

This notification is a concern because it suggests we used to successfully analyze this binary but now cannot, because we are (for the first time) attempting to location a PDB for it. This could indicate that your rule's CanAnalyze needs to be filtering something it is not. In any case, the new appearance of E_PDB_NO_DEBUG_INFO is probably something we shouldn't see in your baseline updates.